src/components/VMenu/VMenu.js

```diff
@@ -202,6 +199,10 @@ export default {
       }
     }
 
+    if (this.$slots.activator) {
+      data.style = { display: this.fullWidth ? 'block' : 'inline-block' }
```

Comment:
One more thing (sorry, I should have noticed it before...) - `.v-menu` has `display: inline-block` by default (https://github.com/ebisbe/vuetify/blob/78417419ac61139170602b05c193f5f2728b327a/src/stylus/components/_menus.styl#L4), so it's enough to set the style only when `this.fullWidth && this.$slots.activator`, and that could be done by using `.v-menu--block` class, so
```js
const data = {
  ...
  class: {'v-menu--block' : this.fullWidth && this.$slots.activator},
  ...
}

Reply:
Oh, forgot about that. Block will have to be the default if there's no activator.